editor/plugins/cpu_particles_3d_editor_plugin.cpp

```diff
@@ -55,6 +55,8 @@ void CPUParticles3DEditor::_notification(int p_notification) {
 void CPUParticles3DEditor::_menu_option(int p_option) {
 	switch (p_option) {
 		case MENU_OPTION_CREATE_EMISSION_VOLUME_FROM_NODE: {
+			emission_tree_dialog->get_scene_tree()->set_marked(node, true);
+			emission_tree_dialog->get_scene_tree()->set_selected(nullptr);
 			emission_tree_dialog->popup_scenetree_dialog();
```

Comment:
With these three methods all being required for the desired behavior in the PR, I wonder if it's worth clumping them up into a single function.

Perhaps by passing what they need as parameters in `popup_scenetree_dialog()`? If getting the `get_scene_tree()` is even possible from there.